test/NuGet.Core.Tests/NuGet.Packaging.Test/PackageArchiveReaderTests.cs

```diff
@@ -1868,7 +1868,32 @@ private static byte[] CreateNonEmptyZipArchive()
         }
 #endif
 
-        private static string ExtractFile(string sourcePath, string targetPath, Stream sourceStream)
+
+        [Fact]
+        public void CanVerifySignedPackages_Platform()
+        {
+            // Arrange
+            using (var test = TestPackagesCore.GetPackageContentReaderTestPackage())
+            {
+                var packageArchiveReader = new PackageArchiveReader(test);
+                // Act
+                var result = packageArchiveReader.CanVerifySignedPackages(null);
+                // Assert
+#if IS_DESKTOP
+                // Verify package signature when running in full frameworks
+                if (RuntimeEnvironmentHelper.IsWindows || !RuntimeEnvironmentHelper.IsMono)
```

Comment:
this condition also seems a bit weird to me. In what case will you not be on mono but also not be on Windows. Should the `||` be a `&&`?

Reply:
Totally right. We can only verify packages only on Windows *and* when we are not running on Mono